src/main/java/net/snowflake/ingest/streaming/internal/FlushService.java

```diff
@@ -314,7 +315,7 @@ void distributeFlushTasks() {
       }
 
       // Distribute work at table level, create a new blob if reaching the blob size limit
-      while (itr.hasNext() && totalBufferSize <= MAX_BLOB_SIZE_IN_BYTES) {
+      while (itr.hasNext() && totalBufferSize <= maxBlobSizeInBytes) {
```

Comment:
Why this change if `maxBlobSizeInByes` is always initialized to the constant? Did you mean to use something from `owningClient` instead?

Reply:
I repurposed my branch that took parameters from the server (including this one) and made some mistakes in the cleanup, including this